### PR TITLE
Support linter workflows

### PR DESCRIPTION
**Pull request by sanyavertolet**
### What's done:
- [x]  Added linters.yml to configure flake8, black and mypy workflow
- [x]  Added test file main.py

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def hello(name):
```
Comment (orchestr7) [inline src/main.py]: use my style:
https://github.com/akuleshov7/h-yapf

Comment (sanyavertolet) [reply src/main.py]: Thank you! 
If only you could give a guide how to add it to my workflow :)

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+      - name: Install Python dependencies
+        run: pip install flake8 mypy black
+
+      - name: Run flake8
+        uses: wearerequired/lint-action@v1
+        with:
+          flake8: true
+          black: true
+          mypy: true
```
Comment (dgreflex) [inline .github/workflows/linters.yml]: Do we need mypy? I think he can get in a lot of trouble.

Comment (sanyavertolet) [reply .github/workflows/linters.yml]: Ok, will remove it

Comment (sanyavertolet) [reply .github/workflows/linters.yml]: Done

**Review by dgreflex — APPROVED** [review]
LGTM

**Pull request merged**